### PR TITLE
MGDAPI-4412 - prepare release v0.41.0

### PR DESCRIPTION
**Pull request by adam-cattermole**
## Overview

[MGDAPI-4412](https://issues.redhat.com/browse/MGDAPI-4412)

- Changed `build_tool` to `docker` as we encountered issues with `podman` version during last release

### CODE REVIEW

File: bundles/0.41.0/metadata/annotations.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: rhmi-cloud-resources
+  operators.operatorframework.io.bundle.channels.v1: rhmi
+  operators.operatorframework.io.bundle.channel.default.v1: rhmi
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.21.0
```
Comment (KevFan) [inline bundles/0.41.0/metadata/annotations.yaml:9]: Minor but think CRO is on `1.14.0` of operator sdk so maybe it's best to rebuild using this version for consitency :thinking:

Comment (adam-cattermole) [reply bundles/0.41.0/metadata/annotations.yaml]: Sure, good shout - I've made the change and the only difference is the 2 locations switched from `operator-sdk-1.21.0` -> `operator-sdk-v1.13.0+git`

```
operator-sdk-1.14.0 version
operator-sdk version: "v1.14.0", commit: "78f08b4852faf344ad3ef457c54f86087aaa0a0a", kubernetes version: "1.21", go version: "go1.16.9", GOOS: "linux", GOARCH: "amd64"
```

**Comment by adam-cattermole** [comment]
These CRDs need regenerating once MGDAPI-4355 has been merged
/hold

**Review by austincunningham — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**